### ISSUE TITLE
Paint background color

### ISSUE THREAD

**Issue by piegamesde** [jesseduffield/lazygit#1327]
**Is your feature request related to a problem? Please describe.**
When using terminal sharing applications like tmate or upterm, the people connected might have different terminal background colors. Thus, it will be unusably ugly for some of the users. This is because lazygit's background is "transparent", i.e. it uses the terminal's background color instead of the theme's.

**Describe the solution you'd like**
Please paint the background color instead of leaving it up to the terminal. Or maybe at least add an option/CLI flat.

**Describe alternatives you've considered**
Alternatively, a theme that works with both light and dark background (is this even possible?) might do it as well.

**Comment by mjarkk** [comment]
Do you know about the light theme option in lazygit?: https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md#light-terminal-theme

**Comment by piegamesde** [comment]
Yes. But unless the light theme also paints the background with a light color, this won't help my issue.

**Comment by mjarkk** [comment]
I don't think this is currently supported and there have to be made changes to [gocui](https://github.com/jesseduffield/gocui) to make this work.
